weblate/templates/keyboard-shortcuts.html

```diff
@@ -0,0 +1,174 @@
+{% load compress %}
+{% load i18n %}
+{% load static %}
+
+{% block extra_script%}
+{% compress js %}
+<script defer data-cfasync="false" src="{% static 'js/keyboard-shortcuts.js' %}{{ cache_param }}"></script>
+{% endcompress%}
+{% endblock %}
```

Comment:
This way the script is inside `<body>`, is that intentional? Maybe better to embed it directly into `weblate/static/loader-bootstrap.js` because it will be loaded on all the pages anyway?

Reply:
Oh, I didn't know that. That wasn't intentional. I will make it  load from the `head`. 

And  `loader-bootstrap.js` is a huge and complex file. Separating the js client source into `static/js/<feature>/scripts*` instead of one giant file I think is better. It will reduce the mental overhead of introducing new logic and modifying old ones. Also it will be a plus in terms of organization. What do you think?

Reply:
`{% block %}` from an included file doesn't override the block from parent (and in case it would, it would cause issues with `{% block extra_script %}` elsewhere. I'd simply include the script in `base.html` if you want to keep it separate.

Reply:
Uh! I see. Alright I'll go with that.